src/hypertable.c

```diff
@@ -1996,6 +1996,41 @@ ts_hypertable_distributed_create(PG_FUNCTION_ARGS)
 	return ts_hypertable_create_internal(fcinfo, true);
 }
 
+/* Go through columns of parent table and check for column data types. */
+static void
+ts_validate_basetable_columns(Relation *rel)
+{
+	int attno;
+	TupleDesc tupdesc;
+
+	tupdesc = RelationGetDescr(*rel);
+	for (attno = 1; attno <= tupdesc->natts; attno++)
+	{
+		Form_pg_attribute attr = TupleDescAttr(tupdesc, attno - 1);
+		Oid typid = attr->atttypid;
```

Comment:
It looks like you're missing a check for dropped columns. When a column is dropped in a table, it is just marked as dropped but not really removed, so you need to check for this case.

Reply:
> It looks like you're missing a check for dropped columns. When a column is dropped in a table, it is just marked as dropped but not really removed, so you need to check for this case.

Fixed this comment by checking for dropped columns.